batchflow/models/torch/losses/binary.py

```diff
@@ -191,3 +191,23 @@ def forward(self, prediction, target):
         sensitivity = (squared_error * inverse).sum() / (inverse.sum() + self.eps)
 
         return self.r * specificity + (1 - self.r) * sensitivity
+
+
+
+class BalancedWeightedBCE(nn.Module):
```

Comment:
`CrossEntropyLoss` from `batchflow.models.torch.losses.core.py` already can do all of that. Maybe, make inherit BCE / make it a special case of it somehow?